flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_NewWaypoint.js

```diff
@@ -1,25 +1,233 @@
 class CDUNewWaypoint {
-    static ShowPage(mcdu) {
+    /**
+     * Callback when a new waypoint has been created, or aborted
+     * @name NewWaypointDoneCallback
+     * @function
+     * @param {WayPoint | undefined} waypoint the resultant new waypoint, or undefined if aborted
+     */
+    /**
+     * New Waypoint Page
+     * @param {A320_CDU_MainDisplay} mcdu
+     * @param {NewWaypointDoneCallback} doneCallback callback when the user is finished with the page
+     * @param {any} _inProgressData private data used by the page
+     */
```

Comment:
**question:** Why are those jsdocs stacked ?

Reply:
One defines a type/name... since this is not TS (😢), the second one uses the type... or at least I think it does 😆